pymc3/distributions/dist_math.py

```diff
@@ -73,16 +74,15 @@ def bound(logp, *conditions, **kwargs):
     """
 
     # If called inside a model context, see if bounds check is disabled
-    try:
-        from pymc3.model import modelcontext
-
-        model = modelcontext(kwargs.get("model"))
-        if not model.check_bounds:
-            return logp
-    except TypeError:  # No model found
-        pass
-
-    broadcast_conditions = kwargs.get("broadcast_conditions", True)
+    if model is not None:
+        try:
+            from pymc3.model import modelcontext
+
+            model = modelcontext(model)
+            if not model.check_bounds:
+                return logp
+        except TypeError:
+            pass # no model found
```

Comment:
```suggestion
    try:
        from pymc3.model import modelcontext

        model = modelcontext(None)
        if not model.check_bounds:
            return logp
    except TypeError:
        pass # no model found
```

Reply:
It works for me. I've committed these changes to the branch. Let me know what you think.

Reply:
Cool, let me know when you do

Reply:
@ricardoV94 just committed the changes. Thanks for your help :thumbsup: